azure-pipelines.yml

```diff
@@ -0,0 +1,59 @@
+trigger:
```

Comment:
Hadn't considered the pipeline yaml in our github repos before, not sure if something to be concerned about. Thoughts @ajhall @debben ?

Reply:
As long as we control all the branches in this repo it *should* be OK... but I think we should segregate our GitHub repos from our internal build agents just for safety. A malicious pipeline that somehow makes it onto any branch in our repos would run on a build agent our internal network, and we shouldn't leave that possibility open.

Reply:
I'll remove it for now